x/programs/runtime/utils.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package runtime
+
+var None = []byte{0}
```

Comment:
None is a prefix, it should not be a `[]byte`. You should only be constructing the `[]byte` where it's needed, not here.

Reply:
I replaced the value of `None` by 0. Also replaced `var` to a `const`.